frontend/src/components/Wheel/index.css

```diff
@@ -0,0 +1,30 @@
+.circle {
+  width: 25em;
+  height: 25em;
```

Comment:
Why are we using `em`s ? And why are we using them here if in other places we use `px`s?